docs/guides/gutenberg-blocks.md

```diff
@@ -86,7 +58,20 @@ _Note: Since the Gutenberg SDK is still being actively developed, the developmen
 
     If you use the paid version, this will allow you to have your own URL that you can use consistently, without having to reconnect.
 
+## Suggestions
+
+* Setting these might be useful for debugging:
+
+  ```php
+  define( 'SCRIPT_DEBUG', true );
+  define( 'GUTENBERG_DEVELOPMENT_MODE', true );
+  ```
+
+  If you use Jetpack-Docker, you could add these to `docker/wordpress/wp-config.php`
```

Comment:
In other places of our documentation, we recommend using an mu-plugin instead:
https://github.com/Automattic/jetpack/tree/master/docker#must-use-plugins-directory

Should we recommend the same here?

Reply:
We found that `SCRIPT_DEBUG` needs to be set earlier, which is why we're recommending `wp-config.php` here.

p1536745828000100-slack-luna